### PR TITLE
Debian initscript improvements

### PR DESCRIPTION
**Pull request by joshuaboniface**
Major set of improvements to the initscripts for Debian builds:

1. Remove `/etc/jellyfin.conf` - this was redundant and it is normal to edit `/etc/default/jellyfin` instead.
2. Set the data path based on `/etc/default/jellyfin`
3. Fix up all 3 init types to use the new executable path with more options.

Also turn off `dotnet build` again as it's not needed.

### CODE REVIEW

**Comment by joshuaboniface** [comment]
OK should be good, tested a couple of the things I changed and it all seems to work - needs one more review!

**Comment by joshuaboniface** [comment]
I have *not* tested the SysV or Upstart scripts though!

File: debian/conf/jellyfin [hunk 1]
```diff
@@ -1,37 +1,36 @@
...
+#
+# SysV init/Upstart options
+#
+
+# Application username
+JELLYFIN_USER="jellyfin"
+# .NET Core runtime binary
+JELLYFIN_DOTNET="/usr/bin/dotnet"
+# Full application command
+JELLYFIN_COMMAND="$JELLYFIN_EXE -programdata $JELLYFIN_DATA -restartpath $JELLYFIN_RESTART_SCRIPT $JELLYFIN_ADD_OPTS"
```
Comment (JustAMan) [inline debian/conf/jellyfin]: This looks dangerous, what if any variable has a space in it?..

File: debian/jellyfin.init [hunk 2]
```diff
@@ -9,13 +9,14 @@
 ### END INIT INFO
 
 # Carry out specific functions when asked to by the system
-pid=`ps -fA|grep dotnet|grep JellyfinServer|awk '{print $2}'| tr -d '\n'`
+pid=`ps -fA|grep dotnet|grep jellyfin|awk '{print $2}'| tr -d '\n'`
```
Comment (JustAMan) [inline debian/jellyfin.init]: This should probably be grepping for `$JELLYFIN_EXE` (and config could be sourced earlier)

File: debian/jellyfin.service [hunk 3]
```diff
@@ -1,12 +1,14 @@
...
 
 [Service]
-User=jellyfin
-ExecStart=/usr/bin/dotnet /usr/lib/jellyfin/bin/EmbyServer.dll
-Restart=on-abort
-TimeoutSec=20
+Type = simple
+EnvironmentFile = /etc/default/jellyfin
+User = jellyfin
+ExecStart = /usr/bin/dotnet ${JELLYFIN_EXE} -programdata ${JELLYFIN_DATA} -restartpath ${JELLYFIN_RESTART_SCRIPT} ${JELLYFIN_ADD_OPTS}
```
Comment (JustAMan) [inline debian/jellyfin.service]: Again what about spaces? And why not utilize `$JELLYFIN_COMMAND`?

File: debian/conf/jellyfin (hunk 1 above)
Comment (JustAMan) [reply debian/conf/jellyfin]: Well then, let's make a note somewhere in debian-specific place that those must not contain unescaped spaces (like, in the config with default values) and be done with it for now.

File: debian/jellyfin.init (hunk 2 above)
Comment (JustAMan) [reply debian/jellyfin.init]: That "original script" was mine I think... It was hacked in like 5 minutes, not something to be kept intact :)

**Pull request merged**